src/app/globals.css

```
@@ -6,272 +1,24 @@
+  --font-sans: var(--font-raleway);
+  --font-serif: var(--font-playfair-display);
 }
 
-sub,
-sup {
-  position: relative;
-  font-size: 75%;
-  line-height: 0;
-  vertical-align: baseline;
+@plugin "daisyui" {
+  themes: night --default;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)